core/templates/domain/utilities/url-interpolation.service.ts

```diff
@@ -43,7 +43,7 @@ export class UrlInterpolationService {
     private utilsService: UtilsService) {}
 
   get DEV_MODE(): boolean {
-    return Constants.DEV_MODE;
+    return AppConstants.DEV_MODE;
```

Comment:
Should we place all these mode getters in one place?

Reply:
These getters are used in order to make mocking in tests possible, I think it would be nice to do this in a different way but I cannot find any other solution.